tests/risk/test_risk_period.py

```diff
@@ -18,459 +18,341 @@
 import numpy as np
 import pytz
 
-from itertools import chain
-from six import itervalues
-
 import zipline.finance.risk as risk
 from zipline.utils import factory
 
 from zipline.finance.trading import SimulationParameters
 from zipline.testing.fixtures import WithTradingEnvironment, ZiplineTestCase
-from . import answer_key
-from . answer_key import AnswerKey
 
-ANSWER_KEY = AnswerKey()
+RETURNS_BASE = 0.01
+RETURNS = [RETURNS_BASE] * 251
 
-RETURNS = ANSWER_KEY.RETURNS
+BENCHMARK_BASE = 0.005
+BENCHMARK = [BENCHMARK_BASE] * 251
+DECIMAL_PLACES = 8
 
 
 class TestRisk(WithTradingEnvironment, ZiplineTestCase):
 
     def init_instance_fixtures(self):
         super(TestRisk, self).init_instance_fixtures()
 
-        start_date = datetime.datetime(
-            year=2006,
-            month=1,
-            day=1,
-            hour=0,
-            minute=0,
-            tzinfo=pytz.utc)
-        end_date = datetime.datetime(
-            year=2006, month=12, day=31, tzinfo=pytz.utc)
-
+        self.start_date = datetime.datetime(2006, 1, 1, tzinfo=pytz.utc)
```

Comment:
Since this test had been written, we have more or less standardized on using `pd.Timestamp` instead of `datetime.datetime`.